### PR TITLE
Classes api implementation

### PR DESCRIPTION
**Pull request by aromakh**

### CODE REVIEW

File: api/swagger.yaml [hunk 1]
```diff
@@ -794,6 +844,159 @@ paths:
...
+              $ref: "#/definitions/Classroom"
+      x-swagger-router-controller: "Classroom"
+  /classroom:
+    post:
+      tags:
+      - "classroom"
+      summary: "Create classroom"
+      operationId: "classroomPOST"
+      produces:
+      - "applciation/json"
```
Comment (william-gooch) [inline api/swagger.yaml]: Should be `application/json`.

Comment (william-gooch) [reply api/swagger.yaml]: Also quite a few of the same typo from here on, just find and replace `applciation` with `application`.

**Review by william-gooch — CHANGES_REQUESTED** [review]
Not a change, but I think it would be a good idea to add the re-using examples code at the bottom of this page: https://swagger.io/docs/specification/adding-examples/  
Seems to allow us to write examples without needing lots of lines to re-use the same brick example many times.

File: api/swagger.yaml (hunk 1 above)
Comment (aromakh) [reply api/swagger.yaml]: @william-gooch  typo is fixed.

**Pull request merged**